okta-hosted-login/package.json

```diff
@@ -7,9 +7,11 @@
     "colors": "^1.1.2",
     "cross-env": "^5.1.3",
     "dotenv": "^8.1.0",
+    "history": "^5.0.0",
     "react": "^16.8.0",
     "react-dom": "^16.8.0",
-    "react-router-dom": "^5.0.0",
+    "react-router": "^6.0.0-beta.0",
```

Comment:
Looks like we will need `history`, but `react-router` has been used as a dependency internally. https://github.com/ReactTraining/react-router/blob/v6.0.0-beta.0/packages/react-router-dom/package.json#L12